### PR TITLE
[Relay, Topi] [Frontend][TFLite, MXNet] ReverseSequence operator

### PR DESCRIPTION
**Pull request by maheshambule**
Ref: https://www.tensorflow.org/api_docs/python/tf/reverse_sequence

Reverse op's compute def is refactored and is now being invoked as a special case of RevereseSequence op.

@jwfromm, @FrozenGene, @kevinthesun, @siju-samuel @mbaret  @u99127 Could you please review?

### CODE REVIEW

**Comment by maheshambule** [comment]
cc: @FrozenGene, @masahi, @yongwww

**Comment by maheshambule** [comment]
ping @FrozenGene, @masahi Please take a look.

**Comment by dhruvaray** [comment]
@anijain2305 - Please review

**Comment by anijain2305** [comment]
@siju-samuel Pinging for review if interested :)

**Comment by maheshambule** [comment]
@siju-samuel, rebased to master. PTAL.

File: tests/python/frontend/tflite/test_forward.py [hunk 1]
```diff
@@ -901,79 +859,6 @@ def test_all_resize():
     if 'RESIZE_NEAREST_NEIGHBOR' in dir(BuiltinOperator()):
         _test_resize(tf.image.resize_nearest_neighbor, data, align_corners=False)
 
-#######################################################################
-# Range
-# -----
-def _test_range(start, limit, delta):
```
Comment (siju-samuel) [inline tests/python/frontend/tflite/test_forward.py:907]: _test_range why removed?  i think this is lost during resolving conflict.

File: tests/python/frontend/tflite/test_forward.py [hunk 2]
```diff
@@ -1679,48 +1564,6 @@ def test_forward_squeeze():
     _test_squeeze(np.arange(6).reshape((2, 1, 3, 1)), [1, 3])
 
 
-#######################################################################
-# Quantize/DeQuantize
-# -------------------
-
-def _test_quantize_dequantize(data):
```
Comment (siju-samuel) [inline tests/python/frontend/tflite/test_forward.py:1686]: _test_quantize_dequantize also missed during conflict resolve?

File: tests/python/frontend/tflite/test_forward.py [hunk 3]
```diff
@@ -2498,9 +2362,10 @@ def test_forward_mediapipe_hand_landmark():
     test_forward_stridedslice()
     test_forward_depthtospace()
     test_forward_spacetodepth()
+    if package_version.parse(tf.VERSION) >= package_version.parse('1.14.0'):
```
Comment (siju-samuel) [inline tests/python/frontend/tflite/test_forward.py:2365]: Move this check inside `test_forward_reverse_sequence`

File: tests/python/relay/test_op_level3.py [hunk 4]
```diff
@@ -663,6 +663,73 @@ def verify_reverse(dshape, axis):
...
+               [[0, 1, 2], [12, 13, 14], [6, 7, 8]],
+               [[18, 19, 20], [3, 4, 5], [24, 25, 26]]],
+              [[[36, 37, 38], [48, 49, 50], [42, 43, 44]],
+               [[27, 28, 29], [39, 40, 41], [33, 34, 35]],
+               [[45, 46, 47], [30, 31, 32], [51, 52, 53]]]]
+    verify_reverse_sequence(indata, [2, 3, 2], 2, 1, np.array(result))
+
+    indata = np.array(np.arange(0, 16)).reshape([4, 4]).astype("int32")
+    result = []
+    with pytest.raises(Exception) as exexcinfo:
```
Comment (siju-samuel) [inline tests/python/relay/test_op_level3.py:726]: exexcinfo > execinfo

File: topi/tests/python/test_topi_transform.py [hunk 5]
```diff
@@ -289,6 +290,85 @@ def check_device(device):
...
+               [[0, 1, 2], [12, 13, 14], [6, 7, 8]],
+               [[18, 19, 20], [3, 4, 5], [24, 25, 26]]],
+              [[[36, 37, 38], [48, 49, 50], [42, 43, 44]],
+               [[27, 28, 29], [39, 40, 41], [33, 34, 35]],
+               [[45, 46, 47], [30, 31, 32], [51, 52, 53]]]]
+    verify_reverse_sequence(indata, [2, 3, 2], 2, 1, np.array(result))
+
+    indata = np.array(np.arange(0, 16)).reshape([4, 4]).astype("int32")
+    result = []
+    with pytest.raises(Exception) as exexcinfo:
```
Comment (siju-samuel) [inline topi/tests/python/test_topi_transform.py:366]: exexcinfo > execinfo

File: tests/python/frontend/tflite/test_forward.py (hunk 1 above)
Comment (maheshambule) [reply tests/python/frontend/tflite/test_forward.py]: Fixed. Got removed during resolving conflicts.

**Comment by maheshambule** [comment]
@siju-samuel, Thanks. Fixed as per the review comments.

**Comment by anijain2305** [comment]
@siju-samuel Thanks for the review. Can you please look again?

**Comment by anijain2305** [comment]
Thanks @siju-samuel @dhruvaray @cbalint13 @maheshambule  this is merged

**Pull request merged**